Makefile

```diff
@@ -18,6 +18,9 @@ test.e2e:
 test.e2e.single:
 	go test -v ./e2e -run '/${name}' -count=1
 
+test.e2e.docker: build.examples
```

Comment:
This prerequisite should be placed for the `test.e2e` target. For now, to avoid the CI error below, just add it only to `test.e2e.docker` target.
https://github.com/tetratelabs/proxy-wasm-go-sdk/runs/2941488799